bundles/org.openhab.binding.surepetcare/src/main/java/org/openhab/binding/surepetcare/internal/handler/SurePetcareBridgeHandler.java

```diff
@@ -0,0 +1,229 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
```

Comment:
Please update the copyright dates on all of your files.
```suggestion
 * Copyright (c) 2010-2021 Contributors to the openHAB project
```

Reply:
done